src/theory/quantifiers/sygus/synth_verify.cpp

```diff
@@ -110,7 +110,17 @@ Result SynthVerify::verify(Node query,
   }
   Trace("sygus-engine") << "  *** Verify with subcall..." << std::endl;
   query = rewrite(query);
-  Result r = checkWithSubsolver(query, vars, mvs, d_subOptions, d_subLogicInfo);
+  Result r;
+  if (options().quantifiers.sygusVerifyTimeoutWasSetByUser)
```

Comment:
```suggestion
  if (options().quantifiers.sygusVerifyTimeout != 0)
```
This captures the actual intention, I guess: The user may explicitly set it to zero, or the default may become something else at some point.

Reply:
Good point!